<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SEAdenitaCoreSettingsGUI</class>
 <widget class="QWidget" name="SEAdenitaCoreSettingsGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cmbScaffold">
       <item>
        <property name="text">
         <string>m13mp18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>p7249</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="1" colspan="2">
      <widget class="QPushButton" name="btnSetPathNtthal">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QCheckBox" name="chkInterpolateDim">
       <property name="text">
        <string>Interpolate Dimensions</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QCheckBox" name="chkOverlay">
       <property name="text">
        <string>Show overlay</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QCheckBox" name="chkAutoScaffold">
       <property name="text">
        <string>Automatically set the scaffold sequence</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="0">
      <widget class="QLineEdit" name="lineNtthal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="3">
      <widget class="QDialogButtonBox" name="dialogButtons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QCheckBox" name="chkXODisplay">
       <property name="text">
        <string>Display possible crossovers</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Scaffold</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Path to ntthal:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="lineCustomScaffold">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QRadioButton" name="rdbDaedalushMesh">
       <property name="text">
        <string>Daedalus mesh model</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QRadioButton" name="rdbIterativeMesh">
       <property name="text">
        <string>Iterative Mesh Model</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnSetPathNtthal</sender>
   <signal>released()</signal>
   <receiver>SEAdenitaCoreSettingsGUI</receiver>
   <slot>onSetPathNtthal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSetPathNtthal()</slot>
 </slots>
</ui>
